src/rgw/rgw_rados.h

```diff
@@ -2595,10 +2582,6 @@ class RGWRados
 
   int create_pool(const rgw_pool& pool);
 
-  /**
-   * create a bucket with name bucket and the given list of attrs
-   * returns 0 on success, -ERR# otherwise.
-   */
   int init_bucket_index(RGWBucketInfo& bucket_info, int num_shards);
```

Comment:
@joscollin @cbodley The comment was introduced by 62e11112398 

```
-
+/**
+ * create a bucket with name bucket and the given list of attrs
+ * returns 0 on success, -ERR# otherwise.
+ */
 int RGWRados::create_bucket(std::string& id, std::string& bucket, map<nstring, bufferlist>& attrs)
 {
```

For current [method prototype](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_rados.h#L2615) , the comment is outdated. I'd prefer to drop this. What do you think?